packages/react/documentation/coding-conventions.md

```diff
@@ -36,7 +36,7 @@ Subcomponents (e.g. `Grid.Cell`) are kept in separate files (e.g. `GridCell.tsx`
 
 ## Text for screen readers only
 
-Use [Visually Hidden](https://designsystem.amsterdam/?path=/docs/components-containers-visually-hidden--docs) or the `ams-visually-hidden` utility class to hide content from sighted users but keep it accessible to non-visual user agents, such as screen readers.
+Use the `ams-visually-hidden` [utility class](http://designsystem.amsterdam/?path=/docs/utilities-css-visually-hidden--docs) to hide content from sighted users but keep it accessible to non-visual user agents, such as screen readers.
 
 Do not use `aria-label`. Tools for automatic translation often [do not translate its value](https://adrianroselli.com/2019/11/aria-label-does-not-translate.html), and it may get overlooked when doing manual translation.
```

Comment:
You could mention that `aria-labbeledby` is recommended to connect visually hidden tekst to a certain element.

Reply:
I was confusing this with [another comment](https://github.com/Amsterdam/design-system/pull/1366#discussion_r1684243095) where aria-label is replaced by a visually hidden field which is not automatically linked to what you're labelling.

Reply:
So this can be resolved?